client/src/ProfessorClasses/ProfessorClasses.js

```diff
@@ -0,0 +1,88 @@
+import React  from 'react'; 
+
+
+class ProfessorClasses extends React.Component{
```

Comment:
Does this represent one class or multiple? if it's only one class and multiple sections then this should be renamed to `ProfessorClass` and then use a separate component to represent the section.